api/src/main/java/io/opentelemetry/trace/DefaultSpan.java

```diff
@@ -31,19 +32,31 @@
 @Immutable
 public final class DefaultSpan implements Span {
 
+  private static final Random random = new Random();
+  private static final DefaultSpan INVALID = new DefaultSpan(SpanContext.getInvalid());
+
+  /**
+   * Returns a {@link DefaultSpan} with an invalid {@link SpanContext}.
+   *
+   * @return a {@code DefaultSpan} with an invalid {@code SpanContext}.
+   * @since 0.1.0
+   */
+  public static DefaultSpan getInvalid() {
+    return INVALID;
+  }
+
   /**
    * Creates an instance of this class.
    *
-   * <p>Each instance will have a unique and valid {@link SpanId}.
+   * <p>Each instance will have unique and valid {@link TraceId} and {@link SpanId}.
    *
    * @return a {@link DefaultSpan}.
-   * @since 0.1.0
    */
-  public static DefaultSpan create() {
+  static DefaultSpan create() {
```

Comment:
Where is this used?

Reply:
Currently it's only used in unit tests after we remove it from API, though https://github.com/open-telemetry/opentelemetry-java/issues/310#issuecomment-498797101 mentioned we should provide a way to create new `DefaultSpan`s with unique `SpanId`:

> Default will generate a new span ID. There will be built-in random algorithm.

Reply:
That constructor will get a SpanContext, generated by the DefaultSpanBuilder. Not now but probably we should remove this and move everyone to use the ctor that accepts the SpanContext.